_overviews/scala3-migration/scaladoc-settings-compatibility.md

```diff
@@ -0,0 +1,55 @@
+| -doc-external-doc | -external-mappings | Comma-separated list of classpath_entry_path#doc_URL pairs describing external dependencies. | Scala3 implementation provides richer syntax. | <i class="fa fa-check fa-lg"></i> |
+| -jdk-api-doc-base | -external-mappings | URL used to link Java API references. | You can specify jdk via -external-mappings since they are generalized setting. | <i class="fa fa-check fa-lg"></i> |
+| -doc-generator | _ | The fully qualified name of a doclet class, which will be used to generate the documentation | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -doc-root-content | -doc-root-content | The file from which the root package documentation should be imported. | |  <i class="fa fa-check fa-lg"></i> |
+| -implicits | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -implicits-debug | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -implicits-show-all | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-sound-shadowing | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -implicits-hide | _ |  | We don't need this in Scala3 |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-debug | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-path | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams-max-classes | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -diagrams-max-implicits | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-timeout | _ |  | We don't need this in Scala3? | <i class="fa fa-check fa-lg"></i> |
+| -diagrams-dot-restart | _ |  | We don't need this in Scala3? |  <i class="fa fa-check fa-lg"></i> |
+| -author | _ |  | Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
+| -raw-output | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -no-prefixes | _ |  | We don't need this in Scala3 | <i class="fa fa-check fa-lg"></i> |
+| -skip-packages | -skip-packages |  |  | <i class="fa fa-check fa-lg"></i> |
+| -no-link-warnings | _ | | | <i class="fa fa-check fa-lg"></i> |
+| -expand-all-types | _ | | Setting should be removed from Scala2 scaladoc | <i class="fa fa-check fa-lg"></i> |
+| -groups | _ | | ~~Implement grouping as it is in Scala3. #11909~~ Actually we support grouping, e. g. [Future](https://scala3doc.virtuslab.com/pr-master/scala3/api/scala/concurrent/Future.html). Since one is using `@group` to actually group his methods, we should consider whether this flag is necessary. Fixed by [#11965](https://github.com/lampepfl/dotty/issues/11965) | <i class="fa fa-check fa-lg"></i> | 
```

Comment:
What do you mean by

> Implement grouping as it is in Scala3. \#11909

?

I don’t understand the description. Are you saying that grouping always enabled and based on the `@groups` tags, and that the `-groups` flag is not necessary anymore?

Last, I’m not sure about the link, is it a stable URL?

Reply:
Ok, my bad, I copied it recklessly. I'll simply remove any comment from here, it is implemented as it has been in Scala2, one needs to add `-groups` flag to add grouping.